README.md

```diff
@@ -55,17 +55,28 @@ The GLIF CLI maps human readable names to account addresses. Whenever you pass a
 
 `glif <command> <command-args> --from testing-account`<br />
 
+To create a read-only label for an arbitrary address:<br />
+`glif wallet label-account <name> <address>`<br />
+
+Note that if you add a built-in actor's address (`f1/f2/f3`), it will be converted to an `f0` ID Address and encoded into a `0x` EVM address format. `0x` style addresses are used when interacting with smart contracts on the FEVM. Read more about it [here](https://docs.filecoin.io/smart-contracts/filecoin-evm-runtime/address-types/#converting-to-a-0x-style-address).
+
+To list all your accounts, including read-only labeled ones:<br />
+`glif wallet list --include-read-only`
```

Comment:
Is the `--include-read-only` flag needed? I think it would be fine to just make that the default.

Reply:
Also, this might be a good spot to introduce the `glif wallet balance` command.

Reply:
I just think it could get annoying if you've added a lot of accounts that are only read-only